### ISSUE TITLE
Colab TPU Wheel torch_xla 2.0 3.9 whl link whereas Colab's Python uses 3.10.11

### ISSUE THREAD

**Issue by JUSEOK-LEE** [pytorch/xla#4974]
## 📚 Documentation

Hi, first of all, thank you so much for providing such a good library.

There is the command in the Jupyter notebook you created (https://colab.research.google.com/github/pytorch/xla/blob/master/contrib/colab/getting-started.ipynb) to use TPU in Google Colab which is importing the 3.9 wheel file.

`!pip install cloud-tpu-client==0.10 torch==2.0.0 torchvision==0.15.1 https://storage.googleapis.com/tpu-pytorch/wheels/colab/torch_xla-2.0-cp39-cp39-linux_x86_64.whl`

```
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
ERROR: torch_xla-2.0-cp39-cp39-linux_x86_64.whl is not a supported wheel on this platform.
```
However, the python version of Colab is now 3.10.11 when checked with `!python --version`, so importing the 3.9 wheel file does not work anymore.

I was wondering if there will be a 3.10 wheel file link like
`https://storage.googleapis.com/tpu-pytorch/wheels/colab/torch_xla-2.0-cp310-cp310-linux_x86_64.whl` officialy available in the near future and if the corresponding Jupyter docs will change accordingly.

This is my first question, so I'm sure I'm asking a lot, thank you for your understanding.

**Comment by JackCaoG** [comment]
Thanks for raising the issue, it seem like this 3.9-3.10 change happened recently. We will try to compile a whl for python 3.10 soon.

**Comment by JUSEOK-LEE** [comment]
Thank you so much for your quick response. Actually, in response to someone else's question in Issues, I also found a way to build the torch-xla whl file on my own, so I kept trying to build it, both on colab and on my mac, but I kept getting more and more errors, probably due to my lack of skill. So I've turned on notifications for the release, and I'll be able to use the link to the new whl file when it goes up. Thanks again for the answer.

**Comment by JUSEOK-LEE** [comment]
> Thanks for raising the issue, it seem like this 3.9-3.10 change happened recently. We will try to compile a whl for python 3.10 soon.

Thank you so much for your quick response. Actually, in response to someone else's question in Issues, I also found a way to build the torch-xla whl file on my own, so I kept trying to build it, both on colab and on my mac, but I kept getting more and more errors, probably due to my lack of skill. So I've turned on notifications for the release, and I'll be able to use the link to the new whl file when it goes up. Thanks again for the answer.

**Comment by willlllllio** [comment]
Colab upgraded to 3.10 but they currently still offer a fallback to 3.9 for now, described here: https://medium.com/google-colab/colab-updated-to-python-3-10-27eb02daa162

`Using the fallback runtime version temporarily allows access to the Python 3.9 runtime, and will be available until mid-May. This is available from the Command Palette via the Use fallback runtime version command when connected to a runtime. Of note, this setting does not persist across sessions — the command will need to be invoked on each new session.`

**Comment by ManfeiBai** [comment]
Hi, @JackCaoG is that ok to assign this issue to you?

**Comment by JackCaoG** [comment]
Some update: I was able to build a python 3.10 whl but that is incompatible with the native pytorch 2.0 whl due to some ABI issue. Looking into fixing it..

**Comment by JackCaoG** [comment]
In the mean time you can use
```
pip3 install https://storage.googleapis.com/tpu-pytorch/tmp/colab_tmp_whl/torch_xla-2.0.0.dev20230504+colab-cp310-cp310-linux_x86_64.whl https://storage.googleapis.com/tpu-pytorch/tmp/colab_tmp_whl/torch-2.0.0.dev20230504+colab-cp310-cp310-linux_x86_64.whl
```

to unblock yourself, through you won't be able to use library like torchaudio and torchtext until I fixed the ABI build issue.

**Comment by JackCaoG** [comment]
This should be fixed by https://github.com/pytorch/xla/pull/5020, the new whl is ready.

**Comment by JackCaoG** [comment]
pr is merged, now you should see the 3.10 whl for colab. feel free to reopen the issue if you run into other probelms.
